packages/headless/src/features/create-article/create-article-insight-analytics-actions.ts

```diff
@@ -0,0 +1,16 @@
+import {CreateArticleMetadata} from 'coveo.analytics';
+import {
+  InsightAction,
+  makeInsightAnalyticsAction,
+} from '../analytics/analytics-utils';
+import {getCaseContextAnalyticsMetadata} from '../case-context/case-context-state';
+
+export const logCreateArticle = (
```

Comment:
In order to have the ability to dispatch this action from an LWC, this action needs to be included in the Insight Analytics Actions Loader: [insight-analytics-actions-loader.ts](https://github.com/coveo/ui-kit/blob/master/packages/headless/src/features/analytics/insight-analytics-actions-loader.ts)

Reply:
added it, will try it in a quantic component